gold-password-input-strength-meter.html

```diff
@@ -153,7 +153,15 @@
               warning: 'None',
               suggestions: 'None'};
             }
-          }
+          },
+          /**
+           * Contains the current score value of the input.
+           */
+          _score: Number,
+          /**
+          * Min strength metter (used on validate).
+          */
+          minStrengthMetter: String,
```

Comment:
A typo all over your changes `Metter` -> `Meter` and therefore, for the sake of consistency, let's call it `minStrengthMeterScoreLabel`